File: ncarg2d/src/libncarg/autogrph/agdshn.f
C
C $Id: agdshn.f,v 1.6 2008-07-27 00:14:34 haley Exp $
C
C                Copyright (C)  2000
C        University Corporation for Atmospheric Research
C                All Rights Reserved
C
C The use of this Software is governed by a License Agreement.
C
      CHARACTER*16 FUNCTION AGDSHN (IDSH)
C
C The value of this function is the name of the dash pattern numbered
C IDSH - that is to say, the character string 'DASH/PATTERN/n.', where
C n is an integer between 1 and 99, equal to MAX(1,MIN(99,IDSH)).
C
      AGDSHN='DASH/PATTERN/  .'
C
      KDSH=MAX(1,MIN(99,IDSH))
C
      DO 101 I=15,14,-1
        AGDSHN(I:I)=CHAR(ICHAR('0')+MOD(KDSH,10))
        IF (KDSH.LE.9) GO TO 102
        KDSH=KDSH/10
  101 CONTINUE
C
  102 RETURN
C
      END

File: ncarg2d/man/autograph/agdshn.m
.TH AGDSHN 3NCARG "March 1993" UNIX "NCAR GRAPHICS"
.na
.nh
.SH NAME
AGDSHN - 
Provides an easy way to generate the names of parameters in
the group 'DASH/PATTERN.', for use in calls to AGSETC and
AGGETC.
.SH SYNOPSIS
AGDSHN (IDSH)
.SH C-BINDING SYNOPSIS
#include <ncarg/ncargC.h>
.sp
char *c_agdshn (int idsh)
.SH DESCRIPTION
.IP IDSH 12
(an input expression of type INTEGER) is between 1 and
26, inclusive.
.SH C-BINDING DESCRIPTION
The C-binding argument description is the same as the FORTRAN 
argument description.
.SH USAGE
This routine generates parameter names for use in calls
to other routines.
For a complete list of parameters available
in this utility, see the autograph_params man page.
.SH ACCESS 
To use AGDSHN or c_agdshn, load the NCAR Graphics libraries ncarg, ncarg_gks, 
and ncarg_c, preferably in that order.    To get smoother curves, 
drawn using spline interpolation, also load libdashsmth.o.  Or,
you can use the ncargf77 command to compile your program and load 
the above libraries, then, to get smoother curves, use the 
-dashsmth option.
.SH SEE ALSO
Online:
autograph,
autograph_params,
agback,
agbnch,
agchax,
agchcu,
agchil,
agchnl,
agcurv,
aggetc,
aggetf,
aggeti,
aggetp,
aggetr,
agpwrt,
agrstr,
agsave,
agsetc,
agsetf,
agseti,
agsetp,
agsetr,
agstup,
agutol,
anotat,
displa,
ezmxy,
ezmy,
ezxy,
ezy
.sp
Hardcopy:
NCAR Graphics Fundamentals, UNIX Version
.SH COPYRIGHT
Copyright (C) 1987-2009
.br
University Corporation for Atmospheric Research
.br
The use of this Software is governed by a License Agreement.
